app/scripts/factories/VariantFactory.js

```diff
@@ -636,6 +633,26 @@ angular.module('oncokbApp')
             this.ncitName = ncitName;
             this.synonyms = synonyms || [];
         }
+        function ReviewedData(item, mutation, drugs, citations) {
+            return {
+                gene: _.isUndefined(item.gene) ? item.hugoSymbol : item.gene.hugoSymbol,
```

Comment:
Why do we keep item.gene and item.hugoSymbol?

Reply:
The api was returned me item.hugoSymbol directly instead of item.gene, but it's disappeared after I rebased branch and updated sql file. I'll remove item.hugosymbol.